test_schemas/en/test_new_section_enabled_checkbox.json

```diff
@@ -0,0 +1,168 @@
+                                        "options": [
+                                            {
+                                                "label": "Neither",
+                                                "value": "Neither"
+                                            }
+                                        ],
+                                        "type": "Checkbox"
+                                    }
+                                ],
+                                "mandatory": false,
+                                "description": ["This is section 1."],
+                                "id": "section-1-question",
+                                "title": "Which sections do you want to enable?",
+                                "type": "MutuallyExclusive"
+                            }
+                        }
+                    ],
+                    "id": "section-1-group",
+                    "title": "Section 1"
+                }
+            ]
+        },
+        {
+            "id": "section-2",
+            "title": "Section 2",
+            "enabled": {
+                "when": {
+                    "in": [
+                        "Section 2",
+                        {
+                            "source": "answers",
+                            "identifier": "section-1-answer"
+                        }
+                    ]
+                }
+            },
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "id": "section-2-block",
+                            "type": "Question",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "section-2-answer",
+                                        "label": "label 2",
+                                        "mandatory": false,
+                                        "type": "Number"
+                                    }
+                                ],
+                                "description": ["This is section 2."],
+                                "id": "section-2-question",
+                                "title": "Which section is this?",
+                                "type": "General"
+                            }
+                        }
+                    ],
+                    "id": "section-2-group",
+                    "title": "Section 2"
+                }
+            ]
+        },
+        {
+            "id": "section-3",
+            "title": "Section 3",
+            "enabled": [
+                {
+                    "when": [
+                        {
+                            "id": "section-1-answer",
+                            "condition": "contains",
+                            "value": "Section 3"
+                        }
+                    ]
+                }
+            ],
```

Comment:
This is the old rules structure.

Reply:
Apologies, somehow it got into the new schema, fixed now, thanks!